src/ui/OGMessageItemBody/index.tsx

```diff
@@ -98,6 +98,7 @@ export default function OGMessageItemBody({
           url={message?.ogMetaData?.defaultImage?.url || ''}
           alt={message?.ogMetaData?.defaultImage?.alt}
           width="100%"
+          height="280px" // FIXME: Find out better to calculate the size of OG Message
```

Comment:
using this could result in overflow in mobile devices is this ok?

Reply:
`height` won't cause overflow, actually this reduced the height on the mobile